src/css/components/NewsItem.css

```diff
@@ -0,0 +1,56 @@
+.NewsItem {
+  font-family: Helvetica;
+  height: 300px;
+  width: 300px;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+
+  &--featured {
+     height: 300px;
+     width: 500px;
+   }
+}
+
+.TileImage {
+  display: inline-block;
+  width: 100%;
+  height: 210px;
+  background: black;
+  vertical-align:top;
+  overflow: hidden;
+}
+
+.TileDetails {
+  display: inline-grid;
+  padding: 10px;
+  max-width: 100%;
+  max-height: 90px;
+}
+
+.Title {
+  display: table;
+  height: fit-content;
+  grid-column-start: 1;
+}
+
+.h-title {
+  display: inline;
+}
+
+.Description {
+  padding-top: 3px;
+  display: inline;
+  grid-row-start: 2;
+  max-height: 3em;
+  overflow: hidden;
+}
+
+.Date {
+  display: table;
+  height: fit-content;
+  grid-column-start: 2;
+  float: right;
+}
+
+.Image {
```

Comment:
CSS is global so anywhere something is called .Image would get affected. That why we want to prefix it with the component name